hack/local-up-cluster.sh

```diff
@@ -505,11 +504,11 @@ function start_apiserver {
 
     authorizer_arg=""
     if [[ -n "${AUTHORIZATION_MODE}" ]]; then
-      authorizer_arg="--authorization-mode=${AUTHORIZATION_MODE} "
+      authorizer_arg="--authorization-mode=${AUTHORIZATION_MODE}"
```

Comment:
To make the following fixes work correctly:

In ./hack/local-up-cluster.sh line 556:
    ${CONTROLPLANE_SUDO} "${GO_OUT}/hyperkube" apiserver ${authorizer_arg} ${priv_arg} ${runtime_config} \
                                                         ^---------------^ SC2086: Double quote to prevent globbing and word splitting.